README.md.template

```diff
@@ -39,6 +39,11 @@ public class BarberView extends FrameLayout {
     @StyledAttr(value = R.styleable.BarberView_stripeColor, kind = Kind.COLOR)
     public int stripeColor;
 
+    // alternately if you use support annotations, the kind can be inferred
+    // @StyledAttr(R.styleable.BarberView_stripeColor)
```

Comment:
Ah good point, still need to keep the Kind for that case since Kind is used internally too. In that case don't worry about it for now.